modular_citadel/code/modules/client/loadout/_engineering.dm

```diff
@@ -0,0 +1,20 @@
+/datum/gear/ceskirt

```

Comment:
This should really be under `modular_kepler` but all the other loadout shit is here so I wont make you change it